### PR TITLE
Solve of stranger problems

### PR DESCRIPTION
**Pull request by saadman-galib**

### CODE REVIEW

**Review by mdmarufsarker — APPROVED** [review]
:1st_place_medal:

File: beginners/easy/022.check-vowel-or-not/206.saadman-galib.check-vowel-or-not.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+char = input('Put your input here: ')
+
+if char.isalpha():
```
Comment (mdmarufsarker) [inline beginners/easy/022.check-vowel-or-not/206.saadman-galib.check-vowel-or-not.py:3]: Good One

**Review by mdmarufsarker — APPROVED** [review]

File: beginners/easy/023.swap-two-number/206.saadman-galib.swap-two-number.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+a, b = map(int, input("Put your input here: ").split())
+
+a = a ^ b
```
Comment (mdmarufsarker) [inline beginners/easy/023.swap-two-number/206.saadman-galib.swap-two-number.py:3]: can you pls add the explanation of this program in python

**Review by mdmarufsarker — APPROVED** [review]

File: beginners/easy/024.Proper-Leap-Years/206.saadman-galib.Proper-Leap-Years.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+year = int(input("Put your input here: "))
+
+if(year % 4 == 0):
```
Comment (mdmarufsarker) [inline beginners/easy/024.Proper-Leap-Years/206.saadman-galib.Proper-Leap-Years.py:3]: try to use 1 single condition

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

File: beginners/easy/032.valid-triangle(sides)/206.saadman-galib.valid-triangle(sides).py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+sides = []
+
+is_valid = False
+
+for i in range(1, 4):
```
Comment (mdmarufsarker) [inline beginners/easy/032.valid-triangle(sides)/206.saadman-galib.valid-triangle(sides).py:5]: create a video (explanation of this code) and send to our slack channel

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]
thanks for the hard work

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — COMMENTED** [review]
copy this code and submit on codeforces [https://codeforces.com/problemset/problem/4/A](url)

**Review by mdmarufsarker — APPROVED** [review]

File: beginners/medium/001.reverse-a-number/206.saadman-galib.reverse-a-number.py [hunk 5]
```diff
@@ -0,0 +1 @@
+print(input()[::-1])
```
Comment (mdmarufsarker) [inline beginners/medium/001.reverse-a-number/206.saadman-galib.reverse-a-number.py:1]: create a video by explaining this line and send to to strangers channel of slack

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

**Review by mdmarufsarker — APPROVED** [review]

File: beginners/medium/001.reverse-a-number/206.saadman-galib.reverse-a-number.py (hunk 5 above)
Comment (saadman-galib) [reply beginners/medium/001.reverse-a-number/206.saadman-galib.reverse-a-number.py:1]: Ok bhaiya

**Review by mdmarufsarker — COMMENTED** [review]
also explain this code in your video. don't need to record multiple video. a single video is ok

**Review by mdmarufsarker — COMMENTED** [review]
also explain this code in your video

File: beginners/easy/022.check-vowel-or-not/206.saadman-galib.check-vowel-or-not.py (hunk 1 above)
Comment (saadman-galib) [reply beginners/easy/022.check-vowel-or-not/206.saadman-galib.check-vowel-or-not.py:3]: Thanks bhaiya

File: beginners/easy/023.swap-two-number/206.saadman-galib.swap-two-number.py (hunk 2 above)
Comment (saadman-galib) [reply beginners/easy/023.swap-two-number/206.saadman-galib.swap-two-number.py:3]: ok bhaiya

File: beginners/easy/024.Proper-Leap-Years/206.saadman-galib.Proper-Leap-Years.py (hunk 3 above)
Comment (saadman-galib) [reply beginners/easy/024.Proper-Leap-Years/206.saadman-galib.Proper-Leap-Years.py:3]: ok bhaiya

**Pull request merged**